message_definitions/v1.0/development.xml

```diff
@@ -339,6 +339,10 @@
         <param index="6" reserved="true" default="0"/>
         <param index="7" reserved="true" default="NaN"/>
       </entry>
+      <entry value="404" name="MAV_CMD_GENERIC_PAYLOAD_ON_OFF" hasLocation="false" isDestination="false">
+        <description>Allows for ON and OFF control for a generic payload.</description>
+        <param index="1" label="Enable" minValue="0" maxValue="1" increment="1">0: Generic payload OFF, 1: Generic payload ON</param>
+      </entry>
```

Comment:
```suggestion
      <entry value="404" name="MAV_CMD_GENERIC_PAYLOAD_DIGITAL_INPUT" hasLocation="false" isDestination="false">
        <param index="1" label="Index/Pin" minValue="0" maxValue="1" increment="1">Index of Pin/Component</param>
        <param index="2" label="Set Point" minValue="0" maxValue="1" increment="1">0: LOW/OFF 1: HI/ON</param>
      </entry>
```

Is there a world where  we would like a component to be able to advertise multiple binary switches. this could be for like say Arming a parachute and then Deploying a parachute. We could achieve this with indexing the Input/Outputs 

`GENERIC_PAYLOAD_STATUS` could have overall payload status, and then we'd have another message say `GENERIC_PAYLOAD_DIGITAL_OUTPUT` which carries the set point of the index/pin

Reply:
UI for primary/secondary/tertiary switches could be discovered based on `GENERIC_PAYLOAD_DIGITAL_OUTPUT`

Reply:
Hi, this is a great idea. I just changed the naming slightly since I wasn't sure if DIGITAL_INPUT would get the idea across. Reviewed the rest with @amy-chen-skydio. Happy to get your input!